grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -854,12 +854,14 @@ class ShipmentService {
             // Check whether there's any stock in the bin location for the given inventory item
             def quantityOnHand = getQuantityOnHand(origin, shipmentItem.binLocation, shipmentItem.inventoryItem, binLocationRequired)
 			def duplicatedShipmentItemsQuantity = getDuplicatedShipmentItemsQuantity(shipmentItem.shipment, shipmentItem.binLocation, shipmentItem.inventoryItem)
+			def allPendingShipmentsWithProduct = getPendingShipmentItemsWithProduct(origin, shipmentItem.product)
+
             log.info "Shipment item quantity ${shipmentItem.quantity} vs quantity on hand ${quantityOnHand} vs duplicated shipment items quantity ${duplicatedShipmentItemsQuantity}"
 
             log.info("Checking shipment item ${shipmentItem?.inventoryItem} quantity [" +
                     shipmentItem.quantity + "] <= quantity on hand [" + quantityOnHand + "]");
             if (duplicatedShipmentItemsQuantity > quantityOnHand) {
-                shipmentItem.errors.rejectValue("quantity", "shipmentItem.quantity.cannotExceedAvailableQuantity",
+                shipmentItem.errors.rejectValue("", "",
```

Comment:
@DominikaHatala I don't think the empty strings are a good idea here. Why did you do it this way?

Reply:
@jmiranda I added the text back and the error message looks fine.